### PR TITLE
EC-1 Database Configuration

### PR DESCRIPTION
**Pull request by sarojmhrzn**

### CODE REVIEW

File: server/migrations/20160911115944-create-category.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+'use strict';
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    return queryInterface.createTable('Categories', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
```
Comment (shreeramneupane) [inline server/migrations/20160911115944-create-category.js:9]: Can you use UUID?

Comment (sarojmhrzn) [reply server/migrations/20160911115944-create-category.js:9]: for current project we are not using this.

**Comment by SandypHendro** [comment]
:+1:

**Comment by shreeramneupane** [comment]
:+1:

**Pull request merged**